ada_2_fhir-r4/cio/2.0.0/sturen_voorstel_contra_indicatie/fhir_instance/cio-svci-tst-1.3-VCI2.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bundle xmlns="http://hl7.org/fhir">
+   <id value="cio-svci-tst-1.3-VCI2"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-ProposalContraIndication-Bundle"/>
+   </meta>
+   <type value="transaction"/>
```

Comment:
type=transaction and total can't co-exist -> delete total

Reply:
Has been corrected!